### PR TITLE
Connected callback (before render) recipe

### PR DESCRIPTION
**Pull request by albarivas**
### What does this PR do?
Adds a new recipe that compares action attribute with connectedCallback

### What issues does this PR fix or reference?

#<Insert GitHub Issue>

## The PR fulfills these requirements:

[x] Tests for the proposed changes have been added/updated.
[x] Code linting and formatting was performed.

### Functionality Before

<insert gif and/or summary>

### Functionality After

<insert gif and/or summary>

### CODE REVIEW

File: force-app/main/default/classes/CreateRecordPrepopulatedController.cls-meta.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+<ApexClass xmlns="urn:metadata.tooling.soap.sforce.com">
```
Comment (pozil) [inline force-app/main/default/classes/CreateRecordPrepopulatedController.cls-meta.xml:2]: @albarivas I'm curious about this change: any idea about how this landed here?
Looking at other `.cls-meta.xml` files I created recently, I see the `http` version of the `xmlns` attribute.

**Review by pozil — APPROVED** [review]
Approved but see my comments on the `.cls-meta.xml` files.

File: force-app/main/default/classes/CreateRecordPrepopulatedController.cls-meta.xml (hunk 1 above)
Comment (albarivas) [reply force-app/main/default/classes/CreateRecordPrepopulatedController.cls-meta.xml:2]: I found why: https://github.com/forcedotcom/cli/issues/173

I don't mind using one or the other though!

**Review by adityanaag3 — CHANGES_REQUESTED** [review]
Recipe looks good, but this component isn't currently added to any page. I guess #171 must be merged first, and then we add this component to the basics tab?

**Review by adityanaag3 — APPROVED** [review]

**Pull request merged**